Description: Add xi:include to access.xml for local overrides
Author: Markus Frosch <lazyfrosch@debian.org>
Last-Update: 2014-09-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/etc/conf.d/access.xml.in
+++ b/etc/conf.d/access.xml.in
@@ -49,6 +49,10 @@
             </access>
         </defaults>
 
+        <!-- include local defaults -->
+        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="/etc/icinga-web/conf.d/access.local.xml">
+            <xi:fallback></xi:fallback>
+        </xi:include>
 
         <!-- Hosts that can be accessed via the console interface -->
         <hosts>
